src/main/java/org/jabref/model/entry/field/StandardField.java

```diff
@@ -63,9 +64,9 @@ public enum StandardField implements Field {
     IDS("ids", FieldProperty.MULTIPLE_ENTRY_LINK),
     INSTITUTION("institution"),
     INTRODUCTION("introduction", FieldProperty.PERSON_NAMES),
-    ISBN("isbn", "ISBN", FieldProperty.ISBN),
-    ISRN("isrn", "ISRN"),
-    ISSN("issn", "ISSN"),
+    ISBN("isbn", "ISBN", FieldProperty.ISBN, FieldProperty.VERBATIM),
+    ISRN("isrn", "ISRN", FieldProperty.VERBATIM),
+    ISSN("issn", "ISSN", FieldProperty.VERBATIM),
```

Comment:
Eprint as well?

Reply:
Done. I am not sure if many more should be added. EPRINTTYPE, FOLDER, ... Not sure how we handle that all.

Reply:
we should follow the biber /biblatex defs here